packages/sdk-artifact/src/ArtifactClient.ts

```diff
@@ -24,6 +26,21 @@ export class ArtifactClient {
     return await this.artifactsApi.uploadArtifact(fileStream, size, key, contentType)
   }
 
+  async uploadDirectory(key: string, directoryPath: string): Promise<void> {
```

Comment:
What about `downloadDirectory`?

Reply:
We need a backend feature for that (list directory). I skipped it for now.